tests/warp/warp_test.go

```diff
@@ -39,6 +41,12 @@ var (
 	manager = runner.NewNetworkManager(config)
 )
 
+var (
```

Comment:
Could we move these local variables into the `ginkgo.Describe` var block so that we put all of the variables needed throughout the test there rather than using a global variable here?

imo we could either put all of the vars there or here, but we should put them in one place.

Reply:
Agreed, it will definitely make it a lot cleaner